bin/reth/src/node/mod.rs

```diff
@@ -259,6 +272,32 @@ impl Command {
                 .build(fetch_client.clone(), consensus.clone(), db.clone()),
         )
     }
+
+    async fn load_peers(&self, network: &NetworkHandle) -> Result<(), eyre::Error> {
+        let reader = match std::fs::File::open(&self.known) {
+            Ok(file) => std::io::BufReader::new(file),
+            Err(e) if e.kind() == ErrorKind::NotFound => return Ok(()),
+            Err(e) => Err(e)?,
+        };
+        info!(target: "reth::cli", file = ?self.known, "Loading saved peers");
+        let known_peers: Vec<NodeRecord> = serde_json::from_reader(reader)?;
+        known_peers.into_iter().for_each(|record| network.add_peer(record.id, record.udp_addr()));
+        Ok(())
+    }
```

Comment:
I want this to be a standalone function that takes the path as an argument.

also please add docs.

Reply:
Made both standalone, and added missing docs.